src/Microsoft.DotNet.Darc/Darc/Operations/GatherDropOperation.cs

```diff
@@ -410,6 +410,7 @@ public string GetFileShareLocationForReleaseJson(DownloadedBuild build)
     private const string sympkgsFileName = "sympkgs-all.txt";
     private const string packagesSubDir = "packages";
     private const string symPackagesSubDir = "assets/symbols";
+    private const string manifestCategory = "manifest";
```

Comment:
Nit, I think this should be named "core-manifest". At this point we only publish the manifests for packages owned by .net core. If aspnet were to be a future owner though, then we would need a separate list.

Reply:
I would also move this up near the other category lines: https://github.com/dotnet/arcade-services/pull/2868/files#diff-395e55dce4798af458bafd198854d9fa13c3499a3a8486808cf67407f6788476R333

Reply:
Updated